TABTestKit/Classes/Contexts/InteractionContext.swift

```diff
@@ -68,6 +68,10 @@ public extension InteractionContext {
 		XCTAssertTrue(element.underlyingXCUIElement.wait(for: element.value == expectedValue), "Element did not have the right value before timing out! Expected: \(expectedValue), actual: \(element.value)")
 	}
 	
+	func label(of element: Element, is expectedLabel: String) {
+	  XCTAssertTrue(element.underlyingXCUIElement.wait(for: element.label == expectedLabel), "Element did not have the right label before timing out! Expected: \(expectedLabel), actual: \(element.label)")
```

Comment:
Sounds like the indentation is not identical with other methods. I think it's 4 spaces for a tab instead of 2.